### ISSUE TITLE
lint :or symbols in map destructuring

### ISSUE THREAD

**Issue by borkdude** [borkdude/clj-kondo#214]
```
(let [{:keys [:a] :or {x 1}} {}] #_[a x])
```
should give a warning about `x` not being present in the destructured symbols

**Comment by borkdude** [comment]
Fixed with 8e2572044fe3720f44d0279fb8a6c5530cd9e1f7
